src/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -59,6 +59,11 @@
         action = component.get("c.getAddresses");
         action.setParams({ hhId : hhId, listAddrExisting : null });
         action.setCallback(this, function(response) {
+
+		    // tell our visualforce page we are done loading
```

Comment:
The real sin is mixing tabs and spaces... which Developer Console does in the most illogical way with no way to change the behavior.